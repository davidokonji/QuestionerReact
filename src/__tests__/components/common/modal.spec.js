import { shallow } from 'enzyme';
import React from 'react';
import Modal from '../../../components/common/modal';

describe('modal component', () => {
  it('should render modal correctly', () => {
    const wrapper = shallow(<Modal />);

    expect(wrapper).toMatchSnapshot();
  });
});
