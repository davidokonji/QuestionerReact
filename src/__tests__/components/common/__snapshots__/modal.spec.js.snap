// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal component should render modal correctly 1`] = `
<div>
  <Modal
    autoFocus={true}
    backdrop={true}
    backdropTransition={
      Object {
        "mountOnEnter": true,
        "timeout": 150,
      }
    }
    centered={false}
    external={
      <button
        className="close"
        style={
          Object {
            "position": "absolute",
            "right": "15px",
            "top": "15px",
          }
        }
        type="button"
      >
        Ã—
      </button>
    }
    fade={true}
    isOpen={false}
    keyboard={true}
    modalTransition={
      Object {
        "timeout": 300,
      }
    }
    onClosed={[Function]}
    onOpened={[Function]}
    role="dialog"
    zIndex={1050}
  >
    <ModalBody
      tag="div"
    />
  </Modal>
</div>
`;
